# Geographic information abaout a Raster

float64 x_offset
float64 y_offset
uint64 x_width
uint64 y_height
float64 cell_width
# Raster's EPSG Spatial Reference System code
uint64 epsg
